docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/type/string-tensor-unpack-15.rst

```diff
@@ -0,0 +1,133 @@
+.. {#openvino_docs_ops_type_StringTensorUnpack_15}
+
+StringTensorUnpack
+===================
+
+
+.. meta::
+  :description: Learn about StringTensorUnpack-15 - operation which unpacks a batch of strings into three tensors.
+
+**Versioned name**: *StringTensorUnpack-15*
+
+**Category**: *Type*
+
+**Short description**: *StringTensorUnpack* operation transforms a given batch of strings into three tensors - two storing begin 
+and end indices of the strings and another containing the concatenated string data, respectively.
+
+**Detailed description**
+
+Consider an ``input`` string tensor containing values ``["Intel", "OpenVINO"]``.
+
+The operator will transform the tensor into three outputs:
+
+* *begins* = [0, 5]
+    * ``begins[0]`` is equal to 0, because the first string starts at the beginning index.
+    * ``begins[1]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``begins.shape`` is equal to [2], because the ``input`` is a batch of 2 strings.
+
+* *ends* = [5, 13]
+    * ``ends[0]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``ends[1]`` is equal to 13, because length of the string "OpenVINO" is 8, and it needs to be summed up with length of the string "Intel".
+    * ``ends.shape`` is equal to ``[2]``, because the ``input`` is a batch of 2 strings.
+
+* *symbols* = "IntelOpenVINO"
+    * ``symbols`` contains concatenated string data encoded in utf-8 bytes, interpretable using ``begins`` and ``ends``.
+    * ``symbols.shape`` is equal to ``[13]``, because it's the length of concatenated ``input`` strings.
+
+When defining *begins* and *ends*, the notation ``[a, b)`` is used. This means that the range starts with ``a`` and includes all values up to, 
+but not including, ``b``. That is why in the example given the length of "IntelOpenVINO" is 12, but *ends* vector contains 13.
+
+**Inputs**
+  
+* **1**: ``data`` - ND tensor of type *string*. **Required.** 
+
+**Outputs**
+
+* **1**: ``begins`` - ND tensor of non-negative integer numbers of type *int32* and of the same shape as ``data``.
+
+* **2**: ``ends`` - ND tensor of non-negative integer numbers of type *int32* and of the same shape as ``data``.
+
+* **3**: ``symbols`` - 1D tensor of concatenated strings data encoded in utf-8 bytes, of type *u8* and length ``[1]``.
```

Comment:
The info about lenght `[1]` seems to be not relevant for `sybmols` input:
```suggestion
* **3**: ``symbols`` - 1D tensor of concatenated strings data encoded in utf-8 bytes, of type *u8* and size equal to the sum of the lengths of each string from the ``data`` input.
```

Reply:
Fixed seconds before the review, but I applied your version anyway as it sounds better.